assets/js/blocks/product-tag/block.tsx

```diff
@@ -211,11 +187,11 @@ class ProductsByTagBlock extends Component {
 						selected={ currentAttributes.tags }
```

Comment:
I think it also makes sense to add a `@ts-expect-error` comment here. What do you think?

Reply:
For now yes it would make sense... until `ProductTagControl` is converted to tsx. But after https://github.com/woocommerce/woocommerce-blocks/issues/9531 we'll have to remove the comment. 
Should I go ahead and add the comment?

Reply:
Hi @gigitux, Should I go ahead and add the comment?

Reply:
Yes, please